frontends/concrete-python/tests/execution/test_min_max.py

```diff
@@ -0,0 +1,219 @@
+"""
+Tests of execution of minimum and maximum operations.
+"""
+
+import random
+
+import numpy as np
+import pytest
+
+from concrete import fhe
+from concrete.fhe.dtypes import Integer
+from concrete.fhe.values import ValueDescription
+
+cases = [
+    [
+        # operation
+        ("minimum_optimized_x", lambda x, y: (np.minimum(x, y), x + 1)),
+        # bit widths
+        4,
+        4,
+        # signednesses
+        False,
+        True,
+        # shapes
+        (),
+        (),
+        # strategy
+        fhe.MinMaxStrategy.CHUNKED,
+    ],
+    [
+        # operation
+        ("minimum_optimized_x", lambda x, y: (np.minimum(x, y), x - 1)),
+        # bit widths
+        4,
+        4,
+        # signednesses
+        True,
+        False,
+        # shapes
+        (2,),
+        (),
+        # strategy
+        fhe.MinMaxStrategy.CHUNKED,
+    ],
+    [
+        # operation
+        ("maximum_optimized_y", lambda x, y: (np.maximum(x, y), y + 1)),
+        # bit widths
+        4,
+        4,
+        # signednesses
+        True,
+        False,
+        # shapes
+        (),
+        (2, 3),
+        # strategy
+        fhe.MinMaxStrategy.CHUNKED,
+    ],
+    [
+        # operation
+        ("maximum_optimized_y", lambda x, y: (np.maximum(x, y), y - 1)),
+        # bit widths
+        4,
+        4,
+        # signednesses
+        False,
+        True,
+        # shapes
+        (),
+        (),
+        # strategy
+        fhe.MinMaxStrategy.CHUNKED,
+    ],
+]
+cases += [
```

Comment:
why not in the first declaration.

Reply:
First is a constant one, this is a list comprehension. I could have replaced `\ncases +=` with a `+` but this is easier to read IMO